op-chain-ops/cmd/check-delta/check-delta.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Check if a directory path is provided
+# Directory must contain the output of batch_decoder's reassemble command
+if [ -z "$1" ]; then
```

Comment:
```suggestion
if [ -z "${1:-}" ]; then
```

to make the -u suggested above happy. Or you could remove this if statement and use something like:

```
directory_path=${1?Must specify path to directory of batch data}
```

Reply:
Applied at https://github.com/ethereum-optimism/optimism/pull/8417/commits/d43897494911165823c3f6310fa421dd2f6bf83a. Thanks!